feature-configs/deploy/gatekeeper/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gatekeeper-operator-product
+  namespace: openshift-operators
```

Comment:
@mmirecki why we use this namespace?

Reply:
@SchSeba this is the default ns we install to when installing from operatorhub